consensus/istanbul/config_test.go

```diff
@@ -51,8 +52,8 @@ func TestGetConfig(t *testing.T) {
 		t.Errorf("error default config:\nexpected: %v\n", DefaultConfig)
 	}
 
-	config := DefaultConfig
-	config.Transitions = []params.Transition{{
+	config := &DefaultConfig
```

Comment:
is there a new issue here as we store the pointer and then we use the same pointer to config3 and config5?

Reply:
yes, as we are changing the values while i want a copy